### ISSUE TITLE
sourced restart fails for gitcollector container

### ISSUE THREAD

**Issue by carlosms** [src-d/sourced-ce#188]
After a `sourced init orgs bblfsh`, restart fails:
```
sourced restart
Recreating srcd-ymjsznno_redis_1      ... done
Recreating srcd-ymjsznno_metadatadb_1 ... done
Recreating srcd-ymjsznno_bblfsh_1     ... done
Recreating srcd-ymjsznno_postgres_1   ... done
Recreating srcd-ymjsznno_ghsync_1     ... done
Recreating srcd-ymjsznno_bblfsh-web_1 ... done
Recreating srcd-ymjsznno_gitbase_1    ... 
Recreating srcd-ymjsznno_gitcollector_1 ... error

Recreating srcd-ymjsznno_gitbase_1      ... done
ary
Recreating srcd-ymjsznno_sourced-ui_1   ... done

ERROR: for gitcollector  Cannot create container for service gitcollector: Duplicate mount point: /library
```

**Comment by smacker** [comment]
what is your docker-compose version? There was a similar bug that is fixed: https://github.com/docker/compose/pull/5910

I can't reproduce with v1.24.1.

**Comment by carlosms** [comment]
docker-compose version 1.21.2, build a133471

It's fixed after updating it to 1.24.1. I completely forgot that docker-compose is installed downloading a script and they don't have a PPA...
